tests/test_auto_dispute_multiple_ids.py

```diff
@@ -0,0 +1,215 @@
+import asyncio
+import io
+from typing import Optional
+from unittest.mock import mock_open
+from unittest.mock import patch
+
+import async_timeout
+import pytest
+from chained_accounts import ChainedAccount
+from telliot_core.apps.core import TelliotConfig
+from telliot_core.apps.core import TelliotCore
+from web3 import Web3
+
+from tellor_disputables.cli import start
+
+
+wallet = "0x39E419bA25196794B595B2a595Ea8E527ddC9856"
+
+
+def txn_kwargs(w3: Web3) -> dict:
+    return {
+        "gas": 500000,
+        "gasPrice": w3.eth.gas_price,
+        "nonce": w3.eth.get_transaction_count(wallet),
+        "from": wallet,
+    }
+
+
+eth_query_id = "0x83a7f3d48786ac2667503a61e8c415438ed2922eb86a2906e4ee66d9a2ce4992"
```

Comment:
Can these hard-coded long query id and query data strings be replaced with just importing the SpotPrice and EVMCall query types? Something like:
```py
eth_query_id, eth_query_data = SpotPrice("eth", "usd").query_id, ...
```